Jenkinsfile

```diff
@@ -15,7 +15,7 @@ pipeline {
     GO_VERSION = '1.16.4'
   }
   options {
-    timeout(time: 3, unit: 'HOURS')
+    timeout(time: 6, unit: 'HOURS')
```

Comment:
wow...

Reply:
compile clang and LLVM remembers me to compile a Linux kernel in a 486 😄

Reply:
finally, it takes about 2h on the CI so I will keep the 3 hour timeout
```suggestion
    timeout(time: 3, unit: 'HOURS')
```